tests/ha/test_server_pod_failure.py

```diff
@@ -0,0 +1,506 @@
+
+    @classmethod
+    def setup_class(cls):
+        """Setup operations for the test file."""
+        LOGGER.info("STARTED: Setup Module operations.")
+        cls.csm_user = CMN_CFG["csm"]["csm_admin_user"]["username"]
+        cls.csm_passwd = CMN_CFG["csm"]["csm_admin_user"]["password"]
+        cls.num_nodes = len(CMN_CFG["nodes"])
+        cls.username = []
+        cls.password = []
+        cls.host_master_list = []
+        cls.node_master_list = []
+        cls.hlth_master_list = []
+        cls.host_worker_list = []
+        cls.node_worker_list = []
+        cls.ha_obj = HAK8s()
+        cls.deploy_lc_obj = ProvDeployK8sCortxLib()
+        cls.s3_clean = cls.test_prefix = cls.random_time = None
+        cls.s3acc_name = cls.s3acc_email = cls.bucket_name = cls.object_name = None
+        cls.restore_pod = cls.deployment_backup = cls.deployment_name = cls.restore_method = None
+        cls.restore_node = cls.node_name = cls.deploy = None
+        cls.restore_ip = cls.node_iface = cls.new_worker_obj = cls.node_ip = None
+        cls.mgnt_ops = ManagementOPs()
+        cls.system_random = secrets.SystemRandom()
+        cls.motr_obj = MotrCoreK8s()
+
+        for node in range(cls.num_nodes):
+            cls.host = CMN_CFG["nodes"][node]["hostname"]
+            cls.username.append(CMN_CFG["nodes"][node]["username"])
+            cls.password.append(CMN_CFG["nodes"][node]["password"])
+            if CMN_CFG["nodes"][node]["node_type"] == "master":
+                cls.host_master_list.append(cls.host)
+                cls.node_master_list.append(LogicalNode(hostname=cls.host,
+                                                        username=cls.username[node],
+                                                        password=cls.password[node]))
+                cls.hlth_master_list.append(Health(hostname=cls.host,
+                                                   username=cls.username[node],
+                                                   password=cls.password[node]))
+            else:
+                cls.host_worker_list.append(cls.host)
+                cls.node_worker_list.append(LogicalNode(hostname=cls.host,
+                                                        username=cls.username[node],
+                                                        password=cls.password[node]))
+
+        cls.rest_obj = S3AccountOperations()
+        cls.s3_mp_test_obj = S3MultipartTestLib(endpoint_url=S3_CFG["s3_url"])
+        cls.test_file = "ha-mp_obj"
+        cls.test_dir_path = os.path.join(TEST_DATA_FOLDER, "HATestMultipartUpload")
+        cls.multipart_obj_path = None
+
+    def setup_method(self):
+        """Following function steps will be invoked prior to each test case."""
+        LOGGER.info("STARTED: Setup Operations")
+        self.random_time = int(time.time())
+        self.restore_node = False
+        self.restore_ip = False
+        self.deploy = False
+        self.s3_clean = {}
```

Comment:
Can you declare it as dict()?

Reply:
> Can you declare it as dict()?

This gives pylint warning if dict() is used  Consider using {} instead of dict()  (use-dict-literal)

Reply:
I don't get this warning, which pylint version are you using?

Reply:
Latest from requirements.txt pylint==2.12.2